docs/migration.md

```diff
@@ -737,11 +737,11 @@ receivers:
     start_at: end
     operators:
     ## Extract timestamp into timestamp field using regex
-    ## rel: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.106.1/pkg/stanza/docs/operators/regex_parser.md
+    ## rel: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.108.0/pkg/stanza/docs/operators/regex_parser.md
     - type: regex_parser
       regex: (?P<timestamp>^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3} (\+|\-)\d{4})
```

Comment:
is this regex change needed

Reply:
i think this was auto changed when running the script? 🤔

Reply:
oh this is the markdown, should be fine